docs/mendel.md

```diff
@@ -0,0 +1,30 @@
+# nf-core/configs: MENDEL Configuration
+
+All nf-core pipelines have been successfully configured for use on the MENDEL CLUSTER at the Gregor Mendel Institute (GMI).
+
+To use, run the pipeline with `-profile conda,mendel`. This will download and launch the [`mendel.config`](../conf/mendel.config) which has been pre-configured with a setup suitable for the MENDEL cluster. A Conda environment will be created automatically and software dependencies will be downloaded from ['bioconda'](https://bioconda.github.io/).
+
+Theoretically, using `-profile singularity,mendel` would download a docker image containing all of the required software, and convert it to a Singularity image before execution of the pipeline. However, there is a regression in the Singularity deployment on MENDEL which renders containers downloaded from public repositories unusable because they lack the /lustre mountpoint.
+
+If you want to run the pipeline containerized anyway you will have to build the image yourself (on a machine where you have root access) using the provided `Singularity` file in the pipeline repository:
+
+```bash
+cd /path/to/pipeline-repository
+echo 'mkdir /lustre > Singularity'
```

Comment:
I think this will overwrite the existing `Singularity` file, no? You presumably want `>>` instead..

Reply:
oops, Good catch! Thanks.